tiup/tiup-component-cluster-start.md

```diff
@@ -16,6 +16,15 @@ tiup cluster start <cluster-name> [flags]
 
 ## Options
 
+### --init
```

Comment:
FYI: In [Step 7. Start a TiDB cluster](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup#step-7-start-a-tidb-cluster), this method is described as safe start and some descriptions are similar, especially the note.